TC-Charter.md

```diff
@@ -9,19 +9,19 @@ easily visible to outsiders.
 
 ## Section 2. Evolution of ISIS Software Project Governance.
 
-Most large, complex open source communities have both a business and a
-technical governance model. ISIS’s technical leadership
-is the Technical Committee (“TC”). ISIS’s business
-leadership is comprised of the USGS ASC Management Team.
-
-This Technical Committee Charter reflects carefully constructed balanced roles
-for the TC and the ASC Management Team in the governance of the ISIS Software
-Package. The charter amendment process is the means by which the TC
-proposes changes to this charter using simple majority of the full TC, the
-proposed changes being subject to review and approval by the ASC Management
-Team. The ASC Management Team may additionally make amendments to the TC
-charter at any time, though the ASC Management Team will not interfere with
-day-to-day discussions, votes or meetings of the TC.
+Most large, complex open source projects have a community of motivated 
+contributors and a set of supporting policies that describe how the
+community strives to operate (e.g. the technical governance model).
+Technical leadership for the ISIS project is the Technical Committee (TC).  
+The ISIS project's contributor community are all of those participants making 
+substantive contributions.
+
+The TC's role is to set policy, carry out those responsibilities detailed in this
+Charter, and potentially mediate charged or contentious concerns between contributors. 
```

Comment:
Should this say "carry out those responsibilities detailed in section 5 of this Charter,"?

Reply:
The charter isn't really that long, I'm not sure that we need a specific pointer.  It is also possible that we might accidentally describe other corollary responsibilities in other parts of this document, and this phrasing covers them all.